info/flow_and_pressure.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: default
+title: Understanding Pressure and Flow in Espresso
+parent: Information
+nav_order: 4
+---
+
+# Understanding Pressure and Flow in Espresso  
+
+The most important thing to understand when thinking about pressure in espresso is that **input flow + resistance = pressure & output flow**, meaning pressure in espresso is a result of the finely ground coffee resisting water flow through it. This water flow can come from a pump, piston, or any hybrid of the two (see: Uniterra nomad). The more flow is supplied, the more pressure can be built given less resistance. Let us start by examining a pump machine.   
+
+## Pump machines  
+
+In a pump machine, the pump supplies a maximum amount of flow sans resistance (known in the espresso world as "water debit").  A vibration pump will output 8mL/s, and a rotary pump will output 30mL/s or more, even. These numbers somewhat represent an upper-bound of the flow rate that can be supplied to make espresso (far faster than anyone would want to brew, even for a "turbo").  So, how do we get to 9 bar? In short, we build pressure by adding resistance, eg. finely ground coffee.  A pump will provide maximum flow until the system builds pressure, and then it will stop at 9 bar or whichever pressure is specified. The mechanism which determines the maximum pressure of a system is called an overpressure valve (OPV). The OPV will reach a setpoint of pressure, and divert the excess flow of the pump in order to maintain that pressure, meaning that the pump is no longer inputting its maximum flow to the coffee puck. On a commercial rotary pump, the OPV is located in between the inlet and outlet of the pump, while on a home machine the OPV is located just after the outlet of the pump. This distinction is important because the measured static pressure of the system will vary depending on where the gauge is located. This small difference can make the communication of brew-pressure between home and commercial users unclear, as a commercial user may "adjust the pump pressure", which would in term make less sense to a home-user where the pump pressure gauge readout occurs before the OPV in the hydraulic circuit and thus will read "9 bars", even if the pressure is lower at the grouphead.  
```

Comment:
> and then it will stop at 9 bar or whichever pressure is specified.

change to stop at a pressure that can support the given flow rate or the maximum pressure specified by the system (OPV).

> and thus will read "9 bars", even if the pressure is lower at the grouphead.

this isn't always the case right? Say the OPV is set to 9 bars but you only have a 6 bar grouphead pressure, the OPV is basically inactive then and the pump pressure gauge should also read 6 bar?

Reply:
Your clarifying second comment is definitely correct.